COLLABORATOR_GUIDE.md

```diff
@@ -285,30 +278,153 @@ npm owner ls video.js
 ```
 
 If you are a core committer, you can request access to npm from one of the current owners.
+Access is managed via an [npm organization][npm org] for [Video.js][vjs npm].
 
 #### GitHub personal access token
 
 This is used to make a GitHub release on videojs. You can get a token from the [personal access tokens](https://github.com/settings/tokens) page.
 
 After generating one, make sure to keep it safe because GitHub will not show the token for you again. A good place to save it is Lastpass Secure Notes.
 
+### Deciding what type of version release
+
+Since we follow the [conventional changelog conventions][conventions],
+all commits are prepended with a type, most commonly `feat` and `fix`.
+If all the commits are fix or other types such as `test` or `chore`, then the release will be a `patch` release.
+If there's even one `feat`, the release will be a `minor` release.
+If any commit has a `BREAKING CHANGE` footer, then the release will be a `major` release.
+Most common releases will be either `patch` or `minor`.
+
 ### Doing a release
 
-To do a release, check out the master branch
+It is also recommended you have a clean clone of Video.js for each release line you want to release.
+That means having a folder for master/v6 and one for 5.x.
+This is because 5.x and 6.x have different versions expecations for release process and have different dependencies.
+Plus, during development you could end up with a dirty repo, so, it just usually easier if you have a clean release repo.
+
+```sh
+# for v6
+git clone git@github.com:videojs/video.js.git videojs-6-release
+# for v5
+git clone git@github.com:videojs/video.js.git videojs-5-release
+```
+
+#### Video.js 6
+
+Make sure go to the master branch and grab the latest updates.
 
 ```sh
 git checkout master
+git pull origin master
 ```
 
-Then run the contrib command to do the next release. Don't forget to provide your GitHub token so the GitHub release goes through.
+At this point, you should run `npm install` because dependencies may have changed.
+
+Then, it's mostly a standard npm package release process with running `npm version`, followed by an `npm publish`.
 
 ```sh
-VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=my-personal-access-token contrib release next patch
+npm version {major|minor|patch}
 ```
 
-This makes a patch release, you can also do a `minor` and a `major` release.
+Depending on the commits that have been merged, you can choose from `major`, `minor`, or `patch` as the versioning values.
+See [deciding what type of version release section](#deciding-what-type-of-version-release).
```

Comment:
Now would be a good time to optionally run `git show` to verify that version went correctly.